### PR TITLE
Moved to version 0.2.0, added the first set of changelogs

### PR DESCRIPTION
**Pull request by twinkarma**
- changed version to 0.2.0
- Changed CHANGELOG to CHANGELOG.md
- Attempted to add the first changelog entries

### CODE REVIEW

**Comment by twinkarma** [comment]
@davidwilby let me know if this looks sensible to you. We can discuss this in our next meeting.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gate-teamware",
-  "version": "0.1.4",
+  "version": "0.2.0\n",
```
Comment (davidwilby) [inline package.json:3]: Oops looks like this new line character might have been added by the versioning script?

**Review by davidwilby — CHANGES_REQUESTED** [review]
Looks good to me

File: package.json (hunk 1 above)
Comment (twinkarma) [reply package.json:3]: Yes looks like a `.strip()` just needed to be added to the read version string. Will put that in the next commit.

**Pull request merged**